scripts/windows_install_perl.ps1

```diff
@@ -15,4 +15,7 @@ echo "installing perl $perl_vesion"
 7z.exe x "$filepath" -xr!c -oc:\strawberry\$perl_version
 if (!$?) { throw 'cmdfail' }
 
+[Environment]::SetEnvironmentVariable('DEFAULT_PERL_VERSION', $perl_version, 'Machine')
+[Environment]::SetEnvironmentVariable('PATH',  "C:\strawberry\$perl_version\perl\bin;" + [Environment]::GetEnvironmentVariable('PATH', 'Machine'), 'Machine')
+
```

Comment:
That's not right - we want to be able to install multiple versions of perl, doing this assignment in here prevents that...

Reply:
Done.